package.json

```diff
@@ -37,6 +37,7 @@
     "atom-space-pen-views": "^2.0.3",
     "fs-plus": "^2.2.0",
     "fuzzaldrin": "^1.2.0",
+    "split-diff": "https://github.com/mupchrch/split-diff/archive/v0.8.3.tar.gz",
```

Comment:
My concerns with using that package as an explicit dependency is that it's behavior and functionality will become a responsibility of git-plus. Some users will probably want to change configurations or expect it to behave a certain way and the onus is put on this package. Also in the scenario where the user has a newer version of split-diff with some killer feature they love, they won't have that in git-plus.

If we enable the file diffing only when split-diff is already installed, there's no responsibility for how it works and its behavior will align with the user configurations.